x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/nlp/tokenizers/BertTokenizerTests.java

```diff
@@ -166,12 +166,21 @@ public void testPunctuation() {
 
     public void testPunctuationWithMask() {
```

Comment:
Could we verify that neversplit works on something like "this should never split"?

Reply:
You mean a never-split token that contains whitespaces?